docs/introduction/running-acceptance-tests.md

```diff
@@ -22,6 +22,26 @@ php phing tests-acceptance
 *Note: In this step you were using Phing target `tests-acceptance`.
 More information about what Phing targets are and how they work can be found in [Console Commands for Application Management (Phing Targets)](/docs/introduction/console-commands-for-application-management-phing-targets.md)*
 
+### How to watch what is going on in the selenium browser
+By default, Shopsys Framework uses `selenium/standalone-chrome` image for `selenium-server` service which means you are not able to watch what is going on in the selenium browser.
+However, there is a quick solution which allows you to watch the progress of your acceptance tests:
+1. In your `docker-compose.yml`, use `standalone-chrome-debug` image for `selenium-server` service and new settings of ports:
+    ```diff
+      selenium-server:
+    -    image: selenium/standalone-chrome:3.11
+    +    image: selenium/standalone-chrome-debug:3.11
```

Comment:
Would it not be easier to use this image by default?

Reply:
On line 8 in this document there is running sh instead of bash, you can update it

Reply:
fixed in https://github.com/shopsys/shopsys/pull/1251/commits/d5643221383bfaf33eacaf4c490e28646df23841